### ISSUE TITLE
[Deque Analysis] Action/Admin Sidebar is not entirely keyboard accessible

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1170]
> Critical: This issue results in blocked content for people with disabilities, and will definitely prevent them from accessing fundamental features or content. This type of issue puts your organization at risk. Prioritize fixing as soon as possible, within the week if possible. Remediation should be a top priority. 

## Deque Analysis Summary

A total of 3 Critical issues exist in the Action/Admin sidebar, all pertaining to keyboard accessibility:

1. (ID 470074) Under "New" menu, "Community", "Collection" and "Item" link are inaccessible to keyboard.  Strangely, when that menu is open, clicking Tab jumps to "Process".  (It may be because these three entries use Javascript and have an `a` tag without an `href`.)
    * This same issue exists under the "Edit" section, and seemingly for any entry that uses `ds-onclick-menu-item`
2. (IDs 470078, 470093) Many links in the sidebar do NOT include an `href` (which I believe causes issue 1).  All links MUST have an `href`.
    * One way to fix this issue by providing an HREF value (even if JavaScript event handlers are used) for the link. (Note: Ensure that if JavaScript event handlers are used, they allow the link to be activated with the Enter key in addition to mouse click.)
    * e.g. `<a href=quickview.html">Quick view</a>`
    * e.g. `<a href="javascript:void(0);">Quick view</a>`

The full list of these issues (which includes links for more info) can be found at this link (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues?activeTab=dt-issue&page=0&pageSize=100&sortField=ordinal&sortDir=asc&row=9&filter%5Bseverity%5D=4&filter%5Bpage_number%5D=9

## More Information / Tools

* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work

**Comment by tdonohue** [comment]
@artlowel : assigning this to you / your team for analysis.

**Comment by artlowel** [comment]
We'll add hrefs to all links, and ensure the tab order makes sense

This will take an estimated **2 hours** and is best done in the context of #1185

**Comment by tdonohue** [comment]
Sounds good @artlowel .  Adding that estimate & assigning back to you for your team.  Feel free to assign both tickets to the same individual & one PR is perfectly OK.
